go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/jwalterweatherman v1.1.0
 	golang.org/x/sys v0.0.0-20210124154548-22da62e12c0c // indirect
+	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c
 )
 
 replace github.com/princessfruittt/ansitotosca => /home/princessfruitt/GolandProjects/ansitotosca
```

Comment:
There are hardcoded local full paths. Relative paths better
Also seems that old name is used here